### PR TITLE
Coverity Issues

### PR DESCRIPTION
**Pull request by mbelluzzo**
128612 and 128613

### CODE REVIEW

**Comment by mbelluzzo** [comment]
1: @cmarcelo 
2: @tcanabrava

File: src/lib/flow/sol-flow-node-options.c [hunk 1]
```diff
@@ -835,7 +835,7 @@ sol_flow_node_named_options_init_from_strv(
         }
 
         mdesc = NULL;
```
Comment (lpereira) [inline src/lib/flow/sol-flow-node-options.c]: You can safely remove this line

Comment (mbelluzzo) [inline src/lib/flow/sol-flow-node-options.c]: true that, anything else? Can I merge with this change?

**Comment by mbelluzzo** [comment]
Merged with proposed changed.

**Pull request closed without merging**